resolwe_bio/processes/variant_calling/gatk.yml

```diff
@@ -77,12 +81,15 @@
     runtime: polyglot
     language: bash
     program: |
+      pip3 install --user resolwe-runtime-utils
```

Comment:
Couldn't we fork the docker image and add this there? This feels very hackish...